src/back/project/Makefile

```
@@ -45,6 +45,14 @@ test-debug test-release: test-%: build-%
 		((test -t 1 && GTEST_COLOR=1 PYTEST_ADDOPTS="--color=yes" ctest -V) || ctest -V)
 	pep8 --ignore=W191,W504 tests
 
+# Test specific test
+.PHONY: test-debug test-release
+test-debug test-release: test-%: build-%
```

Команда называется так же как предыдущая. Она же затрёт её, разве нет?
```suggestion
.PHONY: test-debug-specific test-release-specific
test-debug-specific test-release-specific: test-%: build-%
```

Добавь тогда пожалуйста такие конструкции и в другие МС

---

Готово.